include/private/element.h

```diff
@@ -25,7 +25,23 @@
 #  include <stumpless/param.h>
 #  include "private/param.h"
 
-#  define FOR_EACH_PARAM_WITH_NAME( ELEMENT, NAME ) \
+/**
+ * Iterates the parameters of an element and executes the following block only
+ * for parameters whose name matches the provided name.
+ *
+ * **Thread Safety: MT-Safe req:element-stable**
```

Comment:
I can't find anything regarding the `req` safety attribute remark in the attributes man page. Where did you get this remark from?